pkg/apis/ceph.rook.io/v1/network_test.go

```diff
@@ -72,10 +72,54 @@ func TestValidateNetworkSpec(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+// test the NetworkSpec.IsHost method with different network providers
+func TestNetworkCephIsHost(t *testing.T) {
+	net := NetworkSpec{HostNetwork: false}
+
+	net.Provider = NetworkProviderHost
+	assert.True(t, net.IsHost())
+
+	net.Provider = NetworkProviderMultus
+	assert.False(t, net.IsHost())
+
+	// test with  nonempty but invalid provider
+	net.Provider = "foo"
+	assert.False(t, net.IsHost())
+
+	// test with  nonempty but invalid provider
+	net.Provider = "foo"
+	assert.False(t, net.IsHost())
```

Comment:
Duplicate?

Reply:
@BlaineEXE  wrote:
> Duplicate?

Thanks for spotting! Fixed.